sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/TrampolineUtil.scala

```diff
@@ -241,4 +241,11 @@ object TrampolineUtil {
   }
 
   def getSparkHadoopUtilConf: Configuration = SparkHadoopUtil.get.conf
+
 }
+
+/**
+ * This class is to only be used when there is no equivalent helper method in
+ * Spark to throw an AnalysisException.
```

Comment:
```suggestion
 * This class is to only be used to throw errors specific to the
 * RAPIDS Accelerator or errors mirroring Spark where a raw
 * AnalysisException is thrown directly rather than via an error
 * utility class (this should be rare).
```